### ISSUE TITLE
Accessibility - semantic HTML

### ISSUE THREAD

**Issue by nicolegooden** [CalebCyphers/Whats-Cookin#48]
Your HTML must be written semantically and ARIA tags should be used (ONLY if needed / appropriate)
